dropshot/src/test_util.rs

```diff
@@ -51,9 +51,9 @@ impl<'a> AllowedHeader<'a> {
     }
 }
 
-// List of allowed HTTP headers in responsees.
+// List of allowed HTTP headers in responses.
 // Used to make sure we don't leak headers unexpectedly.
-const ALLOWED_HEADERS: [AllowedHeader<'static>; 5] = [
+const ALLOWED_HEADERS: [AllowedHeader<'static>; 6] = [
```

Comment:
To make sure I understand, we're explicitly whitelisting headers that can be inserted, right? Does this get used anywhere? For example, it might be good to have `add_response_header` return a `Result`, if one tries to add an unsupported header.

Reply:
This is only used in our tests to validate that the resulting headers are what we expect. I don't think we need to do any validation of headers beyond what `HeaderName` already does for us. That might change in the future; I'm not sure if it's worth making `add_response_header` return a `Result<(), Error>` now -- thoughts?

Reply:
Ah, thanks I missed that. I'm fine with not returning a Result yet, too. Dave suggested exposing the `HeaderMap` directly, which I think skirts a lot of these questions I had about multiple values for headers, invalid names, etc. That sounds like a good approach at this point.